### PR TITLE
(page)footer component

### PR DESCRIPTION
**Pull request by sophiaaachow**
# Description

Please include a summary of the change and where applicable, the issue you are addressing. Please also include relevant motivation and context. List any dependencies that are required for this change. If necessary, please update documentation (found under the `collections` folder) as well.
Add footer component

Fixes # (issue number)

## Type of change

Please check on the checkbox for those that are relevant (put x between the brackets)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Code enhancement and update (non-breaking change)
- [ ] Security patch

## How Has This Been Tested?

Please describe or list the test cases that you ran to verify your changes, and provide instructions so we can reproduce them. 
Ran locally

### CODE REVIEW

**Comment by rav3** [comment]
@sophiaaachow 
Added styling fixes to the footer component

File: collections/_components/footer.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+layout: layout-sidenav
+description: Insert description
+category: mandatory
+react-link: ?path=/docs/components-footer--default-story
+github-link: true
+toc: true
+---
+
+{% capture default-footer %}
+<div class="bg-dark text-light p-4">
```
Comment (rav3) [inline collections/_components/footer.html:12]: @sophiaaachow 
You can take and copy the entire footer HTML snippets from 
_includes/footer.html

**Review by clukhei — COMMENTED** [review]
btw what is assets/downloads/sgds-govtech.zip file for?

**Comment by rav3** [comment]
> btw what is assets/downloads/sgds-govtech.zip file for?

This is from the previous version. We used to deliver the sgds library zip file as an alternative

**Pull request merged**